### PR TITLE
Remove @python2-only third party stubs that aren't also dependencies

### LINKED ISSUE #5969: Remove archived facebook-scribe stubs?
https://github.com/facebookarchive/scribe

The first paragraph of the README says:

> This is an archived project and is no longer supported or updated by Facebook.  Please do not file issues or pull-requests against this repo. If you wish to continue to develop this code yourself, we recommend you fork it.

Does typeshed have a policy on stubs for archived projects? Looking over the [third-party library removal policy](https://github.com/python/typeshed/blob/master/CONTRIBUTING.md#third-party-library-removal-policy), it doesn't mention how to handle obsolete/archived projects, if at all. If this issue is accepted, perhaps that doc could updated too.

Removing scribe would allow removing its dependency fb-303 which is also archived and is not installable via the PyPI project.

### PR DESCRIPTION
**Pull request by jdufresne**
These third party stubs have all had a recent release with their version
updated to match the latest of its upstream package.

The following packages continue to exist as dependencies of other third
party stubs, so they remain:

- enum34
- futures
- ipaddress
- six

Closes #5969

### CODE REVIEW

**Comment by srittau** [comment]
I think this is a good idea, but I'd like more maintainer opinions before merging.

**Comment by hauntsaninja** [comment]
I'm in favour of this

**Comment by Akuli** [comment]
I like this too. I don't have a strong opinion about Python 2 only stubs, but less stubs to maintain is good.

**Review by srittau — APPROVED** [review]

**Review by srittau — APPROVED** [review]

**Pull request merged**
